apps/sms/views/shared/js/navigation.js

```diff
@@ -106,7 +105,7 @@ var Navigation = {
     var currentArgs = currentPanel.args || {};
 
     for (var arg in args) {
-      if (args[arg] !== currentArgs[arg]) {
+      if (args[arg] != currentArgs[arg]) {
```

Comment:
everywhere we use isCurrentPanel and know for sure this is a number.

Another possibility is to use parseInt and keep the number if the result is not NaN... but this has a perf penalty, right ?